core/templates/pages/learner-dashboard-page/goals-tab.component.ts

```diff
@@ -30,6 +30,7 @@ import { WindowDimensionsService } from 'services/contextual/window-dimensions.s
 import { I18nLanguageCodeService } from 'services/i18n-language-code.service';
 
 import './goals-tab.component.css';
+import { i } from 'mathjs';
```

Comment:
I don't think this is the `i` you intended to use in the code. Please fix this.

Reply:
okay I have added index, number also there and everything is working fine and I don't think there's any change required but if you want I can do one thing.
initalise i as =>  i: number and then in the place of i I can use this.i and then also everything will work fine

![Screenshot from 2023-03-09 18-42-56](https://user-images.githubusercontent.com/118007662/224033702-65256a79-286c-43b8-8334-d9abee862272.png)

line no: 70 and 95.

And if not so, then please guide me what should I do because I am clueless.
Thanks!

Reply:
I meant to say that the i that you imported is not a variable used for iterating. It is a constant. It is a complex number (i.e. the square root of `-1` to be specific). Please remove this import.